src/dmd/typinf.d

```diff
@@ -43,7 +45,10 @@ extern (C++) void genTypeInfo(const ref Loc loc, Type torig, Scope* sc)
     {
         if (!global.params.useTypeInfo)
         {
-            .error(loc, "`TypeInfo` cannot be used with -betterC");
+            if (e)
+                .error(loc, "expression `%s` uses the gc and cannot be used with -betterC", e.toChars());
```

Comment:
I'd suggest that GC should be capitalized here.

Reply:
ok